src/core/credrank/markovProcessGraph.js

```diff
@@ -728,5 +776,11 @@ function virtualizedMarkovEdge(
       parameters.gammaBackward
     );
   }
+  if (MarkovEdgeAddress.hasPrefix(address, radiationGadget.prefix)) {
+    const nodeAddress = radiationGadget.fromRaw(address);
+    const index = NullUtil.get(nodeIndex.get(nodeAddress));
+    const probability = radiationTransitionProbabilities[index];
+    return radiationGadget.markovEdge(nodeAddress, probability);
+  }
```

Comment:
Note to self: radiation edges are most common, so we get a slight perf optimization if we check for them first.